src/BulletCollision/Gimpact/btBoxCollision.h

```diff
@@ -234,7 +234,24 @@ ATTRIBUTE_ALIGNED16	(class) btAABB
 
 	btAABB()
 	{}
-
+	
```

Comment:
@erwincoumans:
I found how disable generate swig-wraper for this method. And revert compare operators in GIM_PAIR, GIM_BVH_DATA, GIM_BVH_TREE_NODE, and btAAB.

please review.